xbmc/network/EventServer.cpp

```diff
@@ -186,7 +182,7 @@ void CEventServer::Run()
     CLog::Log(LOGERROR, "ES: Invalid port range specified %d, defaulting to 10", port_range);
     port_range = 10;
   }
-  if (!m_pSocket->Bind(any_addr, m_iPort, port_range))
+  if (!m_pSocket->Bind(!CSettings::GetInstance().GetBool(CSettings::SETTING_SERVICES_ESALLINTERFACES), m_iPort, port_range))
```

Comment:
@paxxi: Take a bit of sleep :-)

2016-07-12 22:26 GMT+02:00 Pär Björklund <notifications@github.com>:

> In xbmc/network/EventServer.cpp
> <https://github.com/xbmc/xbmc/pull/10104#discussion_r70515393>:
>
> > @@ -186,7 +182,7 @@ void CEventServer::Run()
> >      CLog::Log(LOGERROR, "ES: Invalid port range specified %d, defaulting to 10", port_range);
> >      port_range = 10;
> >    }
> > -  if (!m_pSocket->Bind(any_addr, m_iPort, port_range))
> > +  if (!m_pSocket->Bind(!CSettings::GetInstance().GetBool(CSettings::SETTING_SERVICES_ESALLINTERFACES), m_iPort, port_range))
>
> Isn't eventserver used by the various remotes like Kore which aren't
> local?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/xbmc/xbmc/pull/10104/files/12e006dfee5da0f398b01ae317ea13c2a1780c4c#r70515393>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABCfHe8uTzqJltm3j958HjTTavU-3o1iks5qU_iDgaJpZM4JKWO1>
> .
>



-- 
                   Key-ID:     0x1A995A9B
                   keyserver: pgp.mit.edu
==============================================================
Fingerprint: 4606 DA19 EC2E 9A0B 0157  C81B DA07 CF63 1A99 5A9B